compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -276,6 +276,20 @@ class CompilationTests {
     compileFilesInDir("tests/explicit-nulls/run", explicitNullsOptions)
   }.checkRuns()
 
+  // Flexible types tests
+  @Test def flexibleTypesRun: Unit = {
+    implicit val testGroup: TestGroup = TestGroup("flexibleTypesRun")
+    compileFilesInDir("tests/flexible-types/run", flexibleTypesOptions)
+  }.checkRuns()
+
+  @Test def flexibleTypesPos: Unit = {
+    implicit val testGroup: TestGroup = TestGroup("flexibleTypesPos")
+    aggregateTests(
+      compileFilesInDir("tests/flexible-types/pos", flexibleTypesOptions),
```

Comment:
The `pos` tests should be run twice: one positive run with the flag; one negative run without the flag, similar to unsafe-nulls.